src/help/getRenderProperties.js

```diff
@@ -52,12 +52,18 @@ function getRenderProperties(element){
 		};
 	}
 	// If canvas (in node)
-	else if(element.getContext){
+	else if(element && element.getContext){
 		return {
 			element: element,
 			renderer: renderers.CanvasRenderer
 		};
 	}
+	else if(element && typeof element === 'object') {
```

Comment:
What you say is true. But the important part is that it should not be a DOM node since people could send in something that is not supported and encodings should not be set there. 

So this should do,
````javascript
else if(element && typeof element === 'object' && !o.nodeName) {
````
tho